src/test/java/org/justjava/gymcore/controller/BookingControllerTest.java

```diff
@@ -60,6 +60,35 @@ void createBooking_returnsCreatedBooking() throws Exception {
         verify(bookingService).createBooking(booking);
     }
 
+    @Test
+    void createBooking_addsToWaitlist_whenClassIsFull() throws Exception {
+        var member = new User("Waitlisted Member", "waitlist@example.com", UserRole.MEMBER, null);
+        member.setId(3L);
+        var trainer = new User("Trainer", "trainer@example.com", UserRole.TRAINER, null);
+        trainer.setId(2L);
+        var gymClass = new GymClass("Spinning", "Indoor cycling", LocalDateTime.of(2025, 2, 5, 9, 0), LocalDateTime.of(2025, 3, 5, 9, 0), 2, trainer);
+        gymClass.setId(10L);
+        var booking = new Booking(member, gymClass);
+
+        given(bookingService.createBooking(booking)).willReturn(ResponseEntity.ok(new ResponseEntity<>("Class is full. You are now on the waitlist.", HttpStatus.OK)));
```

Comment:
this is failing [on build](https://github.com/frank-connolly/gymcore/actions/runs/13506636584/job/37737621104?pr=44)
![image](https://github.com/user-attachments/assets/43a59026-0252-426a-8c4c-17670a8bddb0)